### DISCUSSION TITLE
The most comprehensive and user-friendly source of article metadata in RIS format

### DISCUSSION THREAD

**Discussion by MrChebur** [asreview/asreview#1520] [Q&A]
What is the most comprehensive and easy-to-use source of metadata about scientific articles that you use to retrieve data in RIS format?

I am currently using [MDPI](https://www.mdpi.com/search?sort=pubdate&page_count=200&featured=&subjects=&journals=&article_types=&countries=) data. The data can be automatically downloaded using browser robots.

I also know about the huge database https://docs.openalex.org. However, if I understand correctly, it does not contain data in RIS format.

**Comment by FelixWdm** [comment]
Hi @MrChebur, it also depends a bit on the what your planning to do. MDPI is limited to their own journals, OpenAlex is much more complete. And is a RIS file needed for the rest of the process?

**Reply by MrChebur** [reply]
@FelixWdm 
Thank you for your response. I use `asreview` to find articles relevant to my research topic. I'm not tied to the RIS format. I can use data in any format as long as `asreview` supports it.

If I understand correctly, all `asreview` users have to find metadata somewhere in a format supported by this software.

How do other `asreview` users accomplish this task?
Obviously, this is the most common task among `asreview` users and has been solved many times.

But I couldn't find an answer to this question in the FAQ and had to reinvent the wheel  - MDPI parsing.
